src/Target.h

```diff
@@ -61,6 +61,8 @@ struct Target {
 
         RegisterMetadata,  ///< Generated code registers metadata for use with halide_enumerate_registered_filters
 
+        Matlab,  ///< Generate a mexFunction compatible with Matlab mex libraries.
```

Comment:
There needs to be a longer description somewhere on how Halide works with matlab. This comment should point to that longer description.